10up/classifai
# Update Settings UI to unify the existing onboarding and settings experience.

### Description of the Change
The PR updates the ClassifAI settings UI to unify the existing onboarding and settings experience.

![Screenshot 2024-12-12 at 5 52 13 PM](https://github.com/user-attachments/assets/348c0713-b6ed-431c-b85c-2ae4b1b8fb5b)


### How to test the Change
1. Activate the plugin and verify that the "Getting Started" notice is displayed.
1. Click on the "Get Started" button, and you will be redirected to the welcome page.
1. Click on "Visit the Settings Dashboard" to access the settings page, or navigate to the settings page via Tools > ClassifAI.
1. Test feature enable/disable functionality and feature settings, and verify that they work as expected.

### Changelog Entry
> Changed -  Updated the settings UI to unify the existing onboarding and settings experience.

### Credits

Props @Sidsector9 [Kate Rickard](https://uk.linkedin.com/in/katerickard) @iamdharmesh 

### Checklist:
